nixos/modules/services/databases/mysql.nix

```diff
@@ -310,8 +304,7 @@ in
     services.mysql.settings.mysqld = mkMerge [
       {
         datadir = cfg.dataDir;
-        bind-address = mkIf (cfg.bind != null) cfg.bind;
-        port = cfg.port;
+        port = 3306;
```

Comment:
how is this merged? maybe `mkDefault` here would make sense?

Reply:
Good catch! Thanks.